test/libyul/functionSideEffects/state.yul

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I suggest either we a) merge these tests into current develop or b) add them in the first commit to see if any of these change. My preference is a).